### ISSUE TITLE
Collection ...collection.json cannot be resolved

### ISSUE THREAD

**Issue by Quraian** [rx-angular/rx-angular#767]
# Description

After updating to Angular 12 on ,y project I got some errors which seem to be coming from @rx-angular/template. I created a new clean Angular 12 project and did `ng add @rx-angular/template`. However, I get thr following error:

```
ℹ Using package manager: npm
✔ Found compatible package version: @rx-angular/template@1.0.0-beta.25.
✔ Package information loaded.

The package @rx-angular/template@1.0.0-beta.25 will be installed and executed.
Would you like to proceed? Yes
✔ Package successfully installed.
An unhandled exception occurred: Collection "C:\\Code\\temp\\node_modules\\@rx-angular\\template\\schematics\\collection.json" cannot be resolved.
See "C:\Users\m4des\AppData\Local\Temp\ng-eh5RUV\angular-errors.log" for further details.
```

# Steps to Reproduce the Issue

* ng new temp --defaults
* cd .\temp\
* ng add @rx-angular/template

# Environment

```
Angular CLI: 12.0.0
Node: 14.16.1
Package Manager: npm 6.14.12
OS: win32 x64

Angular: 12.0.0
... animations, cli, common, compiler, compiler-cli, core, forms
... platform-browser, platform-browser-dynamic, router

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1200.0
@angular-devkit/build-angular   12.0.0
@angular-devkit/core            12.0.0
@angular-devkit/schematics      12.0.0
@schematics/angular             12.0.0
rxjs                            6.6.7
typescript                      4.2.4
```

**Comment by edbzn** [comment]
Hi, it was resolved in #587, it will be released in the next version.

**Comment by BioPhoton** [comment]
Fixed in alpha.9
